modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPRequest.testcase

```diff
@@ -144,6 +144,74 @@ definition {
 		}
 	}
 
+	@description = "LPS-180356 - Verify that  the 'phone number' field is enabled  if it's not registered and it is possible to fill this"
+	@priority = 3
+	test IsPossibleFillPhoneNumberField {
+		task ("Given the Employee user don't have a phone number registered") {
+			ApplicationsMenu.gotoPortlet(
+				category = "User",
+				panel = "Control Panel",
+				portlet = "Users and Organizations");
+
+			EVPUtils.gotoContactInformation(
+				contactUser = "Contact",
+				enterUser = "Employee User",
+				inforUser = "Contact Information");
+
```

Comment:
Could you add the macro: `ApplicationsMenu.gotoPortlet` inside this macro: `EVPUtils.gotoContactInformation` as a first step? Due to this step is needed to set the contact user, so the test could be shortened